### PR TITLE
Add disconnect from slack button

### PR DESCRIPTION
**Pull request by MeijerM1**
- Users can disconnect their Kudo-O-Matic account from Slack

See the corresponding [backend PR](https://github.com/kabisa/kudo-o-matic/pull/27) as well.

### CODE REVIEW

File: src/modules/user/UserPage.tsx [hunk 1]
```diff
@@ -86,15 +106,45 @@ export class UserPage extends React.Component<Props, State> {
                       </a>
                     </span>
                     {data && data.viewer.slackId ? (
-                      <Header data-testid="slack-connected">Your account is connected to slack!</Header>
+                      <Segment data-testid="slack-connected">
+                        <Header>Your account is connected to slack!</Header>
+                        <p>You can disconnect your Kudo-O-Matic account from Slack by clicking the button below.</p>
```
Comment (pascalw) [inline src/modules/user/UserPage.tsx:111]:
```suggestion
                        <p>You can disconnect your Kudo-O-Matic account from Slack using the button below.</p>
```

File: src/modules/user/UserPage.tsx [hunk 2]
```diff
@@ -86,15 +106,45 @@ export class UserPage extends React.Component<Props, State> {
                       </a>
                     </span>
                     {data && data.viewer.slackId ? (
-                      <Header data-testid="slack-connected">Your account is connected to slack!</Header>
+                      <Segment data-testid="slack-connected">
+                        <Header>Your account is connected to slack!</Header>
```
Comment (pascalw) [inline src/modules/user/UserPage.tsx:110]:
```suggestion
                        <Header>Your account is connected to Slack!</Header>
```

File: src/modules/user/UserPage.tsx [hunk 3]
```diff
@@ -86,15 +106,45 @@ export class UserPage extends React.Component<Props, State> {
                       </a>
                     </span>
                     {data && data.viewer.slackId ? (
-                      <Header data-testid="slack-connected">Your account is connected to slack!</Header>
+                      <Segment data-testid="slack-connected">
```
Comment (pascalw) [inline src/modules/user/UserPage.tsx:109]: Maybe you could extract this whole Segment into a separate component or private function for readability? The render function is pretty big now.

Comment (MeijerM1) [reply src/modules/user/UserPage.tsx:109]: Will do!

**Pull request merged**